Formula/libzt.rb

```diff
@@ -0,0 +1,32 @@
+class Libzt < Formula
+  desc "Encrypted P2P networking library for applications"
```

Comment:
Might be worth mentioning `GPL-licensed library` to ensure people don't inadvertently use it in proprietary applications.

Reply:
Good idea, however we provide an option to purchase a commercial license as well. How would you recommend that I signal that in this short description?

Reply:
People will be able to see that from the homepage.

Reply:
Understood, added.